### PR TITLE
events: Add info on vmchecker workshop (July 10, 2023)

### PR DESCRIPTION
**Pull request by razvand**
Add information on vmchecker workshop, takin place online, on Discord, on Monday, July 10, 2023. The workshop is targeted to a technical crowd that would be interested in automating the deployment, checking and grading of assignment submissions.

### CODE REVIEW

File: website/docs/events/workshop-2023-07-10.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Workshop: vmchecker: Automated Assignment Deployment, Checking and Grading
+
+Among the most difficult and time consuming activities in teaching / training are assignments, or practical evaluated items.
```
Comment (teodutu) [inline website/docs/events/workshop-2023-07-10.md]:
```suggestion
Among the most difficult and time-consuming activities in teaching / training are assignments, or evaluated practical items.
```

File: website/docs/events/workshop-2023-07-10.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+An assignment consists of a statement and set of goals.
+Typically, assignments have a grading scale that ensure a fair grading scheme and also provides students / trainees with a set of steps for solving it.
```
Comment (teodutu) [inline website/docs/events/workshop-2023-07-10.md]:
```suggestion
Typically, assignments have a grading scale that ensures a fair grading scheme and also provides students / trainees with a set of steps for solving it.
```

File: website/docs/events/workshop-2023-07-10.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+The deployment, checking and grading of assignments is a candidate for automation.
```
Comment (teodutu) [inline website/docs/events/workshop-2023-07-10.md]:
```suggestion
Deploying, checking and grading assignments are candidates for automation.
```

File: website/docs/events/workshop-2023-07-10.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+Given the large number of submissions, having an automation engine taking care of this improves the work of teachers and also provides immediate feedback and direction to students.
+
+At [OpenEdu](https://open-education-hub.github.io/), we've developed and enhanced [vmchecker](https://github.com/systems-cs-pub-ro/vmchecker-next/wiki/Teaching-Assistant-Handbook), an engine to automate deployment, checking and grading of assignments.
+vmchecker uses modern solutions (GitLab, GitLab CI, Docker) to provide a scalable and efficient infrastructure for assignment automation.
+It has been successfully used at the Faculty of Automatic Control and Computers, in University POLITEHNICA of Bucharest, during the last year, with close to 30,000 submissions being handled.
```
Comment (teodutu) [inline website/docs/events/workshop-2023-07-10.md]:
```suggestion
It has been successfully used at the Faculty of Automatic Control and Computers, in University POLITEHNICA of Bucharest, during the last year, with close to 30,000 submissions being handled by June 2023.
```
You can also point out that this number is due to increase as more classes adopt vmchecker.

File: website/docs/events/workshop-2023-07-10.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+It is currently integrated with the [Moodle instance at UPB](https://curs.upb.ro/).
```
Comment (teodutu) [inline website/docs/events/workshop-2023-07-10.md:14]: Maybe some would like to read more details on this, like how a student / teacher can access vmchecker from Moodle?

File: website/docs/events/workshop-2023-07-10.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
+# Workshop: vmchecker: Automated Assignment Deployment, Checking and Grading
+
+Among the most difficult and time consuming activities in teaching / training are assignments, or practical evaluated items.
+An assignment consists of a statement and set of goals.
+Typically, assignments have a grading scale that ensure a fair grading scheme and also provides students / trainees with a set of steps for solving it.
+The deployment, checking and grading of assignments is a candidate for automation.
+Given the large number of submissions, having an automation engine taking care of this improves the work of teachers and also provides immediate feedback and direction to students.
+
+At [OpenEdu](https://open-education-hub.github.io/), we've developed and enhanced [vmchecker](https://github.com/systems-cs-pub-ro/vmchecker-next/wiki/Teaching-Assistant-Handbook), an engine to automate deployment, checking and grading of assignments.
```
Comment (teodutu) [inline website/docs/events/workshop-2023-07-10.md:9]: Maybe consider moving the repo under the OpenEdu organisation? Not sure if there's anything preventing that, but logically, it's an OpenEdu product, so I guess it should be there.

File: website/docs/events/workshop-2023-07-10.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+vmchecker uses modern solutions (GitLab, GitLab CI, Docker) to provide a scalable and efficient infrastructure for assignment automation.
+It has been successfully used at the Faculty of Automatic Control and Computers, in University POLITEHNICA of Bucharest, during the last year, with close to 30,000 submissions being handled.
+It is currently integrated with the [Moodle instance at UPB](https://curs.upb.ro/).
+
+vmchecker is open source software, similar to other OpenEdu components.
+The aim is to have it available to everyone interested in automating assignments and other practical activities.
+
+We aim to further develop vmchecker based on user feedback.
+Planned items are adding virtual machine support (currently checking is done on containers only), using the same infrastructure for lab tasks (practical items during assisted sessions).
```
Comment (teodutu) [inline website/docs/events/workshop-2023-07-10.md]:
```suggestion
Our planned items are adding virtual machine support (currently checking is done on containers only), and using the same infrastructure for lab tasks (practical items during assisted sessions).
```

File: website/docs/events/workshop-2023-07-10.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+And extending its use to other faculties, universities or interested parties.
+
+As such, on **Monday, July 10, 2023, starting from 15:00 EEST**, we are organizing a **hands-on online workshop** to initiate interested parties in the installing, deployment and use of vmchecker.
+It takes place on the [OpenEdu Hub Discord server](https://bit.ly/OpenEduHub).
+The aim is to have vmchecker included as an automation solution in universities and educational institutions in Romania starting with the next academic year (starting in October 2023).
+
+In this 2.5 hours online workshop, we welcome people with technical familiarity that have an educational / academic involvement and want to improve the teaching/training-related processes in their institution.
+If you like to take a direct part in simplifying the grading of assignments in your class, this workshop is for you.
+We will have a 1 hour tutorial / demo session (that we will also record) followed by another hour where you will do your own vmchecker setup, with assistance from our side.
```
Comment (teodutu) [inline website/docs/events/workshop-2023-07-10.md]:
```suggestion
We will have a 1-hour tutorial / demo session (that we will also record and upload on [the OpenEdu YouTube channel](https://www.youtube.com/@OpenEduHub)) followed by another hour where you will do your own vmchecker setup, with assistance from our side.
```

File: website/docs/events/workshop-2023-07-10.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+We aim to further develop vmchecker based on user feedback.
+Planned items are adding virtual machine support (currently checking is done on containers only), using the same infrastructure for lab tasks (practical items during assisted sessions).
+And extending its use to other faculties, universities or interested parties.
+
+As such, on **Monday, July 10, 2023, starting from 15:00 EEST**, we are organizing a **hands-on online workshop** to initiate interested parties in the installing, deployment and use of vmchecker.
+It takes place on the [OpenEdu Hub Discord server](https://bit.ly/OpenEduHub).
+The aim is to have vmchecker included as an automation solution in universities and educational institutions in Romania starting with the next academic year (starting in October 2023).
+
+In this 2.5 hours online workshop, we welcome people with technical familiarity that have an educational / academic involvement and want to improve the teaching/training-related processes in their institution.
```
Comment (teodutu) [inline website/docs/events/workshop-2023-07-10.md]:
```suggestion
In this 2.5-hour online workshop, we welcome people with technical familiarity that have an educational / academic involvement and want to improve the teaching/training-related processes in their institution.
```

**Review by teodutu — CHANGES_REQUESTED** [review]
I think the announcement does its job well enough. My comments are mostly about details and cosmetics.

File: website/docs/events/workshop-2023-07-10.md (hunk 6 above)
Comment (razvand) [reply website/docs/events/workshop-2023-07-10.md:9]: @teodutu , yes, the repo will be moved. We'll do it after the workshop. Cc: @Sergiu121 , @Jokeswar

File: website/docs/events/workshop-2023-07-10.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
+# Workshop: vmchecker: Automated Assignment Deployment, Checking and Grading
+
+Among the most difficult and time consuming activities in teaching / training are assignments, or evaluated practical items.
+An assignment consists of a statement and set of goals.
+Typically, assignments have a grading scale that ensures a fair grading scheme and also provides students / trainees with a set of steps for solving it.
```
Comment (teodutu) [inline website/docs/events/workshop-2023-07-10.md]:
```suggestion
Typically, assignments have a grading scale that ensures fair grading and also provides students / trainees with a set of steps for solving it.
```

File: website/docs/events/workshop-2023-07-10.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+It has been successfully used at the Faculty of Automatic Control and Computers, in University POLITEHNICA of Bucharest, during the last year, with close to 30,000 submissions being handled by June 2023.
+The number of submissions is expected to grow as new classes adopt vmchecker.
+
+It is currently integrated with the [Moodle instance at UPB](https://curs.upb.ro/).
+A teaching assistant configures a Moodle assignment instance and connects it to a vmchecker assignment repository.
+A student can use Moodle to submit the assignment and view the resulting grade and test output.
+Behind the scenes, Moodle interacts with vmchecker to pass the assignment submission archive and to retrieve the results.
+
+vmchecker is open source software, similar to other OpenEdu components.
+The aim is to have it available to everyone interested in automating assignments and other practical activities.
```
Comment (teodutu) [inline website/docs/events/workshop-2023-07-10.md]:
```suggestion
This makes it available to everyone interested in automating assignments and other practical activities.
```
You mentioned "aim" below and it already is available.

File: website/docs/events/workshop-2023-07-10.md [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+
+We aim to further develop vmchecker based on user feedback.
+Our planned items are adding virtual machine support (currently checking is done on containers only), using the same infrastructure for lab tasks (practical items during assisted sessions).
+And extending its use to other faculties, universities or interested parties.
+
+As such, on **Monday, July 10, 2023, starting from 15:00 EEST**, we are organizing a **hands-on online workshop** to initiate interested parties in the installing, deployment and use of vmchecker.
+It takes place on the [OpenEdu Hub Discord server](https://bit.ly/OpenEduHub).
+The aim is to have vmchecker included as an automation solution in universities and educational institutions in Romania starting with the next academic year (starting in October 2023).
+
+In this 2.5-hours online workshop, we welcome people with technical familiarity that have an educational / academic involvement and want to improve the teaching/training-related processes in their institution.
```
Comment (teodutu) [inline website/docs/events/workshop-2023-07-10.md]:
```suggestion
In this 2.5-hour online workshop, we welcome people with technical familiarity that have an educational / academic involvement and want to improve the teaching/training-related processes in their institution.
```
You'd say "64-bit architecture", not "64-bits".

**Review by teodutu — CHANGES_REQUESTED** [review]
I only left a few suggestions. When those are fixed, I think this can be merged.

**Review by teodutu — APPROVED** [review]
👌🏿

**Pull request merged**